src/models/volunteer_log.ts

```diff
@@ -88,7 +92,14 @@ export const VolunteerLogs: VolunteerLogCollection = {
           'volunteer_activity',
           'volunteer_activity.volunteer_activity_id',
           'volunteer_hours_log.volunteer_activity_id')
-        .select(query.fields ? pick(query.fields, ModelToColumn) : ModelToColumn),
+        .innerJoin(
```

Comment:
Obviously this is fine for now. Ideally in future I'd like to think of a way to get rid of it again though 🙊

Reply:
Yeah, I would also like to separate out the types at some point so that additional fields are not optional on main type declarations.